src/index.js

```diff
@@ -1,20 +1,29 @@
-import * as calendarUtils from "./utils/CalendarUtils/CalendarUtils";
-import * as proxy from "./utils/Proxy/Proxy";
 import * as main from "./Main/Main";
+import defaultProxyServerUrl from "./resources/DefaultProxyServerUrl/DefaultProxyServerUrl";
+import { isDefined } from "./utils/JavaScriptUtils/JavaScriptUtils";
 
-const index = (
+export default (
   container,
-  gitHubUsers = [],
-  gitLabUsers = [],
-  proxyServerUrl = proxy.defaultProxyServerUrl
+  gitHubUsers,
+  gitLabUsers,
+  proxyServerUrl = defaultProxyServerUrl
 ) => {
-  if (calendarUtils.requiredParamsExist(container, gitHubUsers, gitLabUsers)) {
-    main.processParams(container, gitHubUsers, gitLabUsers, proxyServerUrl);
-  } else {
-    throw new Error(
-      "Please provide the required parameters in the appropriate format."
-    );
+  if (!isDefined(container)) {
+    throw new Error("Arguments are not sufficently provided.");
```

Comment:
If it's already broken down to specific cases let's use a specific message.

Reply:
Please do address this still

Reply:
Sure, I'm working on it.

Reply:
I'm just adjusting the tests a bit.

Reply:
How about something like c67168e?